src/fhir/formatters.ts

```diff
@@ -104,7 +104,7 @@ export function formatAge(age: fhir4.Age): string {
 // Formats a string ISO date to MM/DD/YYY.
 export function formatISODateStringToDate(dateStr?: string | number): string | undefined {
   if (!dateStr) return undefined;
-  const date = typeof dateStr === "number" ? new Date(dateStr).toISOString() : dateStr;
+  const date = typeof dateStr === "number" ? new Date(dateStr * 1000).toISOString() : dateStr;
```

Comment:
Do we know this will always be a timestamp that is not properly using ms rather than shorter unix timestamp? This is ugly, but should we do something here like:
```suggestion
  const ensureJsTimestamp = (n: number) => `${n}`.length <= 11 ? n * 1000 : n;
  const date = typeof dateStr === "number" ? new Date(ensureJsTimestamp(dateStr)).toISOString() : dateStr;
```